packages/backend/src/helpers/flow-templates.ts

```diff
@@ -0,0 +1,84 @@
+import type { AppEventKeyPair } from '@plumber/types'
+
+import type {
+  CreateTemplatedFlowInput,
+  FlowConfig,
+} from '@/graphql/__generated__/types.generated'
+import Flow from '@/models/flow'
+import User from '@/models/user'
+
+import logger from './logger'
+
+export const DEFAULT_FLOW_TEMPLATE: CreateTemplatedFlowInput = {
+  flowName: 'Send notifications',
+  trigger: {
+    appKey: 'formsg',
+    eventKey: 'newSubmission',
+  },
+  actions: [
+    {
+      appKey: 'postman',
+      eventKey: 'sendTransactionalEmail',
+    },
+  ],
+  demoVideoId: 'formsg-postman',
+}
+
+async function createFlowFromTemplate(
+  flowName: string,
+  trigger: AppEventKeyPair,
+  actions: AppEventKeyPair[],
+  user: User,
+  isPreCreated: boolean,
+  demoVideoId: string,
+): Promise<Flow> {
```

Comment:
sorry i missed this out. but there are too many parameters for this function. a good yardstick is >3. move to a object param. but not a blocker can do the refactor later when implementing receipts